spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java

```diff
@@ -144,7 +145,7 @@ public void testMultiplePerTargetAspects() throws SecurityException, NoSuchMetho
 		aspect2.setOrder(5);
 		advisors.addAll(
 				getFixture().getAdvisors(new SingletonMetadataAwareAspectInstanceFactory(aspect2, "someBean2")));
-		Collections.sort(advisors, new OrderComparator());
+		advisors.sort(new OrderComparator());
```

Comment:
If optimize, then do one step further :)

> sort(advisors, OrderComparator.INSTANCE);

Reply:
modify to OrderComparator.sort(advisors);